static/js/components/search/WithSearchkitManager.js

```diff
@@ -1,40 +1,37 @@
 // @flow
 /* global SETTINGS: false */
 import React from 'react';
-import R from 'ramda';
 import { SearchkitManager, SearchkitProvider } from 'searchkit';
 
 import { getDisplayName } from '../../util/util';
 import { getCookie } from '../../lib/api';
 
-const withSearchkitManager = R.curry(
-  WrappedComponent => {
-    class WithSearchkitManager extends React.Component {
-      searchkit: SearchkitManager;
+const withSearchkitManager = () => WrappedComponent => {
```

Comment:
For consistency with `withEmailDialog`, but I agree it doesn't make much sense. I'll remove it